src/ebay-menu/README.md

```diff
@@ -26,7 +26,7 @@ Name | Type | Required | Description
 --- | --- | --- | ---
 `type` | String | No | Can be `radio`/`checkbox`
 `checked` | Number | No | when used with `radio` type will check the item with the corresponding index
-`onKeydown` | Function | No | arguments: (index: number, checked: boolean)
+`onKeydown` | Function | No | arguments: (index: number, checked: boolean, event)
```

Comment:
Seems there's inconsistency here. We have param event for `onKeyDown()` but not on `onSelect()` nor `onChange()`. Same thing with eBay-fake-menu, where event was just removed from `onSelect()` in this PR.

On some components (e.g ebay-radio, ebay-select ,ebay-switch, ...), `onChange()` has param `event`. I think make sense **to always return `event` as the first parameter**, which will give more flexibility to the user. Checkout [Material UI](https://mui.com/material-ui/) or [Microsoft Fluent UI](https://developer.microsoft.com/en-us/fluentui#/controls/web) which returns event param as the first one on every handlers.

I would suggest not to remove event param on this PR, and always return `event` for ebay-menu-button. We can create another issue to change event handlers on all components, which will be breaking changes

Reply:
I put it at the very end for backwards compatibility. EbayUI marko has a discrepancy between readme and real code, so I'm not sure what's best...